drydock/patches/kustomization

```diff
@@ -0,0 +1,50 @@
+patches:
+{% if DRYDOCK_NEWRELIC -%}
+- target:
+      kind: Deployment
+      name: lms
+  path: plugins/drydock/k8s/newrelic.yml
+{% endif -%}
+- target:
```

Comment:
If we delete these jobs then you cannot run `tutor do init` right? Can we put this behind a setting and check if leaving the jobs allows us to run `tutor do` commands? For example we can check by running the tutor command to create users.

In fact, maybe we can use an the [`skip` ArgoCD hook](https://argo-cd.readthedocs.io/en/stable/user-guide/resource_hooks/#available-hooks) to simply ignore the jobs when applying instead of deleting them. I think this could make us compatible with the tutor cli.